pkg/driver/node_test.go

```diff
@@ -762,9 +808,9 @@ func TestNodeGetInfo(t *testing.T) {
 	}
 }
 
-func newTestNodeService(metadata cloud.MetadataService, mounter mount.Interface) nodeService {
+func newTestNodeService(cloud cloud.Cloud, mounter mount.Interface) nodeService {
```

Comment:
I think we can just pass in Metadata here since it is changed to be public:

```suggestion
func newTestNodeService(cloud cloud.MetadataService, mounter mount.Interface) nodeService {
```

So that it saves us from creating mock cloud in the test as well.

Reply:
Good catch.  Done and was able to remove all mock cloud creation.